owners/src/owners_check.js

```diff
@@ -168,6 +168,48 @@ class OwnersCheck {
 
     return reviewerSuggestions.join('\n\n');
   }
+
+  /**
+   * Build the check-run comment describing current approval coverage.
+   *
+   * @param {!FileTreeMap} fileTreeMap map from filenames to ownership subtrees.
+   * @return {string} a list of files and which owners approved them, if any.
+   */
+  buildCurrentCoverageText(fileTreeMap) {
+    const allFilesText = Object.entries(fileTreeMap)
+      .map(([filename, subtree]) => {
+        const fileApprovers = this.approvers.filter(approver =>
+          subtree.hasOwner(approver)
+        );
+
+        if (fileApprovers.length) {
+          return `- ${filename} (${fileApprovers.join(', ')})`;
+        } else {
+          return `- [NEEDS APPROVAL] ${filename}`;
+        }
+      })
+      .join('\n');
+
+    return ['=== Current Coverage ===', allFilesText].join('\n\n');
```

Comment:
Why not \`=== Current Coverage ===\n\n${allFilesText}\`?

Reply:
Sure